sql-statements/sql-statement-flashback-to-timestamp.md

```diff
@@ -0,0 +1,103 @@
+---
+title: FLASHBACK [CLUSTER | DATABASE | TABLE] TO TIMESTAMP
+summary: TiDB 数据库中 FLASHBACK [CLUSTER | DATABASE | TABLE] TO TIMESTAMP 的使用概况。
+---
+
+# FLASHBACK [CLUSTER | DATABASE | TABLE] TO TIMESTAMP
+
+TiDB v6.4.0 引入了 `FLASHBACK [CLUSTER | DATABASE | TABLE] TO TIMESTAMP` 语法，其功能是将集群、数据库、数据表的数据恢复到特定的时间点。
+
+## 语法
+
+```sql
+FLASHBACK CLUSTER TO TIMESTAMP '2022-09-21 16:02:50';
+FLASHBACK DATABASE db TO TIMESTAMP '2022-09-21 16:02:50';
+FLASHBACK TABLE tbl1, tbl2 TO TIMESTAMP '2022-09-21 16:02:50';
+```
+
+### 语法图
+
+```ebnf+diagram
+FlashbackToTimestampStmt ::=
+    "FLASHBACK" ("CLUSTER" | "DATABASE" DBName | "TABLE" TableNameList) "TO" "TIMESTAMP" stringLit
+```
+
+## 注意事项
+
+* `FLASHBACK` 指定的时间点需要在 Garbage Collection (GC) life time 时间内。你可以使用系统变量 [`tidb_gc_life_time`](/system-variables.md#tidb_gc_life_time-从-v50-版本开始引入) 配置数据的历史版本的保留时间（默认值是 `10m0s`）。可以使用以下 SQL 语句查询当前的 `safePoint`，即 GC 已经清理到的时间点：
+
+    ```sql
+    SELECT * FROM mysql.tidb WHERE variable_name = 'tikv_gc_safe_point';
```

Comment:
Should we document the variable `tikv_gc_safe_point`?

Reply:
It's not a system variable, just a line in `mysql.tidb` table.